HazelcastCloudTests/csharphazelcastcloudtests/CloudTests/StandardClusterTests.cs

```diff
@@ -0,0 +1,115 @@
+            HazelcastOptions options;
+            if (isSslEnabled)
+            {
+                options = Helper.CreateClientConfigWithSsl(_sslEnabledCluster.NameForConnect, _sslEnabledCluster.Token, isSmart,
+                    _sslEnabledCluster.CertificatePath, _sslEnabledCluster.TlsPassword);
+                _tmpClusterObject = _sslEnabledCluster;
+            }
+            else
+            {
+                options = Helper.CreateClientConfigWithoutSsl(_sslDisabledCluster.NameForConnect, _sslDisabledCluster.Token,
+                    isSmart);
+                _tmpClusterObject = _sslDisabledCluster;
+            }
+
+            var logger = options.LoggerFactory.Service.CreateLogger("test");
+
+            logger.LogInformation("Create client");
+            await using var client = await HazelcastClientFactory.StartNewClientAsync(options);
+            await using var map = await client.GetMapAsync<string, string>("MapForTest");
+            await Helper.MapPutGetAndVerify(map, logger);
+
+            logger.LogInformation("Scale up cluster from 2 node to 4");
+            await RcClient.SetHazelcastCloudClusterMemberCount(_tmpClusterObject.Id, 4);
+            await Helper.MapPutGetAndVerify(map, logger);
+
+            logger.LogInformation("Scale down cluster from 4 node to 2");
+            await RcClient.SetHazelcastCloudClusterMemberCount(_tmpClusterObject.Id, 2);
+            await Helper.MapPutGetAndVerify(map, logger);
+
+            logger.LogInformation("Stop cluster");
+            await RcClient.StopHazelcastCloudCluster(_tmpClusterObject.Id);
+
+            logger.LogInformation("Resume cluster");
+            await RcClient.ResumeHazelcastCloudCluster(_tmpClusterObject.Id);
+
+            logger.LogInformation("Wait 5 seconds to be sure client is connected again");
+            await Task.Delay(5_000);
+
+            await Helper.MapPutGetAndVerify(map, logger);
+            await client.DisposeAsync();
+        }
+
+        [TestCase(true)]
+        [TestCase(false)]
+        [Timeout(30_000)]
+        public async Task TryConnectSslClusterWithoutCertificates(bool isSmart)
+        {
+            var options = Helper.CreateClientConfigWithoutSsl(_sslEnabledCluster.NameForConnect,
+                _sslEnabledCluster.Token, isSmart);
+            options.Networking.Ssl.Enabled = true;
+            options.Networking.ConnectionRetry.ClusterConnectionTimeoutMilliseconds = 10000;
+            var logger = options.LoggerFactory.Service.CreateLogger("test");
+            bool value = false;
+            try
+            {
+                logger.LogInformation("Try connecting ssl cluster without certificates");
+                await using var client = await HazelcastClientFactory.StartNewClientAsync(options);
+            }
+            catch
+            {
+                value = true;
+            }
+            Assert.IsTrue(value, "Client shouldn't be able to connect ssl cluster without certificates");
```

Comment:
Can we use https://docs.nunit.org/articles/nunit/writing-tests/assertions/classic-assertions/Assert.Throws.html here?